src/Init/Data/Int/LemmasAux.lean

```diff
@@ -45,4 +45,31 @@ theorem bmod_neg_iff {m : Nat} {x : Int} (h2 : -m ≤ x) (h1 : x < m) :
   · rw [Int.emod_eq_of_lt xpos (by omega)]; omega
   · rw [Int.add_emod_self.symm, Int.emod_eq_of_lt (by omega) (by omega)]; omega
 
+theorem bmod_eq_iff_of_lt_of_lt {x : Int} {y : Nat} (hge : -y ≤ x * 2) (hlt : x * 2 < y) :
```

Comment:
Isn't this `bmod_eq_of_le_of_lt`? The hypothesis`hge` is an `le`, not an `lt`. I'm also not sure why we need an `iff` in the theorem name.